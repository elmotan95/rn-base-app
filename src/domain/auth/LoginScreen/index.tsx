import {StyleSheet, Text, View} from 'react-native';
import React from 'react';
import {AppContext, TAppContext} from '../../../context/AppContext';

const LoginScreen = ({}) => {
  const {isAuth} = React.useContext(AppContext) as TAppContext;
  return (
    <View style={styles.container}>
      <Text>Login Screen</Text>
      <Text>{'is Auth: ' + isAuth}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
  },
});

export default LoginScreen;
