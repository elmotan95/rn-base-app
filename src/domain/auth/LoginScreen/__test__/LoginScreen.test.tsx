import {render} from '@testing-library/react-native';
import React from 'react';
import AppContextProvider from '../../../../context/AppContext';
import LoginScreen from '../index';

describe('LoginScreen Snapshot test', () => {
  it('should render correctly', () => {
    const tree = render(
      <AppContextProvider>
        <LoginScreen />
      </AppContextProvider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
