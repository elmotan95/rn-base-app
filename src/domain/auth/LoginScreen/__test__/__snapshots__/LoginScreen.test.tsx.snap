// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginScreen Snapshot test should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Login Screen
  </Text>
  <Text>
    is Auth: false
  </Text>
</View>
`;
